hs/src/Daml/Cucumber.hs

```diff
@@ -285,7 +285,9 @@ runTestSuite opts = do
           case shouldRunTests of
             True -> do
               writeDamlScript testFile result
-              (ec, damlTestStdout, damlTestStderr) <- liftIO $ readProcessWithExitCode damlPath ["test", "--files", testFile] ""
+              let damlProc = proc damlPath ["test", "--files", testFile]
```

Comment:
I think it would be better to use the DAML_PROJECT environment variable

Reply:
I did some manual tests as well and it is looking good, thanks!

I built daml-cucumber and from a nix-shell in hs I tested different combinations to make sure it works

```sh
../result/bin/daml-cucumber --source $(pwd) --features $(pwd)/features.feature
../result/bin/daml-cucumber --source . --features ./features.feature`
./result/bin/daml-cucumber --source test --features test/features.feature`
```